File: ProyectoCsharp/Modelo/Gerente.cs
namespace Modelo
{
    public class Gerente : Usuario
    {
        //Constructor
        public Gerente()
        {

        }


        public Gerente(long id_usuario, string nombre, string apellidos, string correo, string dni, string username,
                        string contraseña, string descripcion_permisos) : base(id_usuario, nombre,
                        apellidos, correo, dni, username, contraseña, descripcion_permisos)
        {

        }
    }
}

File: TrabajoJavaLp2/src/modelo/Gerente.java
package modelo;
public class Gerente extends Usuario{
	public Gerente(){
		
	}

	public Gerente(long id_usuario, String nombre, String apellidos, String correo, String codigo,
                    String contrasena, String descripcion_permisos)
    {
		super(id_usuario, nombre,
                    apellidos, correo, codigo, contrasena, descripcion_permisos);
    }
}